_posts/2024-05-28-apache-cxf-toolexception-wsdl2java.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: 'Apache CXF ToolException "Tools plugin provider jaxb context init failed" when using wsdl2java '
+authors: ['markus']
+featuredImage: gitlab-ci-pipelines
+excerpt: "Build a multiproject spanning Pipeline triggered by Tags in Gitlab CI"
+permalink: '/2024/05/28/cxf-jaxb-toolexception-wsdl2java.html'
+categories: [CXF, JAXB, wsdl2java, XML, SOAP, Code Generation, Gradle]
+header:
+  text: Apache CXF ToolException with JAXB / wsdl2java 
+  image: post
+---
+
+Imagine, in a project you are asked to consume a SOAP webservice. Yes, SOAP is aged as stone BUT it is also very percisely specified as a interface, which make it still a common choice when it comes to intergrate systems. And only integrated systems deliver great benefit on the one hand as well as (old) legacy system have proven their business case which is why they are worth to integrate. (There is *gold* in legacy.) 
```

Comment:
I'd actually fully reformulate it as it's a bit hard to follow, but in case you desire to keep it as is, here are some typo corrections.
```suggestion
Imagine, in a project you are asked to consume a SOAP webservice. Yes, SOAP is aged as stone BUT it is also very precisely specified as an interface, which make it still a common choice when it comes to integrate systems. And only integrated systems deliver great benefit on the one hand as well as (old) legacy system have proven their business case which is why they are worth to integrate. (There is *gold* in legacy.) 
```